airbyte-webapp/src/views/Connector/ServiceForm/components/Property/LabelInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import { JSONSchema7Type } from "json-schema";
+import React from "react";
+
+import { TextWithHTML } from "components";
+
+import styles from "./LabelInfo.module.scss";
+
+interface IProps {
+  label: React.ReactNode;
+  examples?: JSONSchema7Type;
+  description?: string;
+}
+
+const LabelInfo: React.FC<IProps> = ({ label, examples, description }) => {
+  const constructExamples = () => {
```

Comment:
This should be a component. I'd argue for moving it above this one, renaming to `Examples` perhaps, and passing it the props so the use should look like:
`<Examples examples={examples} />`

Reply:
I like that suggestion, thanks!